operator/gitops/argocd/pipeline-service/tekton-results/base/kustomization.yaml

```
@@ -50,10 +50,20 @@ patchesJson6902:
           - "/bin/bash"
           - "-c"
           - |
-            if oc get secret -n tekton-results tekton-results-database 2>&1 >/dev/null; then exit; fi
-            oc create secret generic tekton-results-database \
+            if oc get secret -n tekton-results tekton-results-postgres 2>&1 >/dev/null; then
+              exit
+            fi
+            if oc get secret -n tekton-results tekton-results-database 2>&1 >/dev/null; then
+              DATABASE_USER=$(oc get secret -n tekton-results tekton-results-database -o jsonpath="{.data.DATABASE_USER}")
+              DATABASE_PASSWORD=$(oc get secret -n tekton-results tekton-results-database -o jsonpath="{.data.DATABASE_PASSWORD}")
+              oc delete secret -n tekton-results tekton-results-database
+            else
+              DATABASE_USER="tekton"
+              DATABASE_PASSWORD="$(dd if=/dev/urandom bs=20 count=1 2>/dev/null | base64)"
+            fi
+            oc create secret generic tekton-results-postgres \
               --namespace="tekton-results" \
-              --from-literal=DATABASE_USER=tekton \
-              --from-literal=DATABASE_PASSWORD=$(dd if=/dev/urandom bs=20 count=1 2>/dev/null | base64)
+              --from-literal=DATABASE_USER="$DATABASE_USER" \
+              --from-literal=DATABASE_PASSWORD="$DATABASE_PASSWORD"
```

```suggestion
              --from-literal=POSTGRES_USER="$DATABASE_USER" \
              --from-literal=POSTGRES_PASSWORD="$DATABASE_PASSWORD"
```

The literal name has changed as well 

https://github.com/openshift-pipelines/tektoncd-results/blob/7e99606c104ed5867271cdf47e0fdf93a70c41e1/config/api.yaml#L44-L49

---

Fixed